### PR TITLE
Fixing dapps sorting

### LINKED ISSUE #3076: Dapps are not sorted left-to-right anymore
@ngotchac Please address.

parity 2016-11-01 21-55-05

### PR DESCRIPTION
**Pull request by ngotchac**
Fixes #3076

### CODE REVIEW

File: js/src/views/Dapps/dapps.css [hunk 1]
```diff
@@ -25,6 +25,6 @@
 
 .item {
   display: inline-block;
-  margin: 0 0 0.25em;
-  width: 100%;
+  margin: 0.25%;
```
Comment (jacogr) [inline js/src/views/Dapps/dapps.css]: We have the margins at 0.25em on the accounts list, would like to keep consistency throughout

Comment (ngotchac) [inline js/src/views/Dapps/dapps.css]: Okay, fixed (nb: `0.125em` but for each which makes `0.25em` for a two column layout)

**Comment by jacogr** [comment]
Spacing still off (Accounts vs Apps) -

parity 2016-11-02 11-47-52

parity 2016-11-02 11-47-32

**Comment by ngotchac** [comment]
Should be ok now

**Pull request merged**